Codeinwp/otter-blocks
# Add Float mode for Sticky Blocks

Closes #1139.


### Summary


- Add controls

### Screenshots 

#### Controls

Make block active

<img width="200" src="https://user-images.githubusercontent.com/17597852/186899839-e5622483-4ce8-498c-911e-453fb6a3256e.png"/>

There is already an active block on the page

<img width="200" src="https://user-images.githubusercontent.com/17597852/186899742-8d73438a-fa0d-4a15-bef6-a9cd8514c7fd.png"/>


There are multiple active blocks on the page

<img width="200" src="https://user-images.githubusercontent.com/17597852/186899444-f3056536-b48e-4785-af84-b9762b7b2f2e.png"/>



----

### Test instructions


1. Make some blocks sticky.
2. Activate `Always Active` in one of them.
3. The rest should have the option deactivated since only one can always be active.

If you encounter any problems, please post the test instance. Sticky is very sensitive about the setup.


---- 

### Checklist before the final review
